hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/HawkbitCommonUtil.java

```diff
@@ -596,7 +595,7 @@ public static Locale getLocaleToBeUsed(final UiProperties.Localization localizat
      */
     public static void initLocalization(final UI ui, final Localization localizationProperties,
             final VaadinMessageSource i18n) {
-        ui.setLocale(HawkbitCommonUtil.getLocaleToBeUsed(localizationProperties, ui.getSession().getLocale()));
+        ui.setLocale(getLocaleToBeUsed(localizationProperties, ui.getPage().getWebBrowser().getLocale()));
```

Comment:
I think ui.getSession().getLocale() was the right choice here. According to the doc this returns "the preferred locale of the user (...). In most cases it is read from the browser"